src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -3227,13 +3239,19 @@ Map {
   },
   "ProgressIndicator" => Object {
     "defaultProps": Object {
+      "className": null,
       "currentItemId": null,
+      "isClickable": false,
       "isVerticalMode": false,
-      "onClickItem": null,
```

Comment:
Removing props is a breaking change. Can we leave it in, put a deprecation notice, and map it to a new prop?